build.sh

```diff
@@ -93,8 +93,11 @@ cd ../..
 
 echo ""
 echo "Copying extra plugins..."
-cp -r plugins/* ckeditor/plugins/
-
+if [ "$(uname)" == "Darwin" ]; then
+	cp -R plugins/ ckeditor/plugins/
```

Comment:
Also checked git bash for windows and `-R` is also supported there.

I'm +1 for simplifying it to `-R` for all platforms.